x/evm/keeper/state_transition.go

```diff
@@ -181,6 +184,16 @@ func (k *Keeper) ApplyTransaction(tx *ethtypes.Transaction) (*types.MsgEthereumT
 
 	res.Hash = txHash.Hex()
 	logs := k.GetTxLogs(txHash)
+
+	if !res.Failed() {
+		err = k.PostTxProcessing(txHash, logs)
+		if err != nil {
```

Comment:
also add the comments from the ADR
```suggestion
		if err := k.PostTxProcessing(txHash, logs); err != nil {
```

Reply:
done